docs/manhole.md

```diff
@@ -18,6 +18,16 @@ listeners:
 (`bind_addresses` in the above is important: it ensures that access to the
 manhole is only possible for local users).
 
+**Docker Note**
+If you are using docker, please ensure you use the following example;
+
+```yaml
+listeners:
+  - port: 9000
+    bind_addresses: ['0.0.0.0']
+    type: manhole
+```
+
```

Comment:
can you say something about needing to forward the port with `docker run` as well?

Reply:
I've added a simple "example" config that anyone who wants to use manhole and is using docker should clearly understand.